08-projects.Rmd

```diff
@@ -4260,6 +4260,8 @@ The following questions will use the dataset found in Scholar:
 
 `/class/datamine/data/lahman/lahman.db`
 
+This is the Lahman Baseball Database. You can find its documentation [here](http://www.seanlahman.com/files/database/readme2017.txt), including the definitions of the tables and columns.
```

Comment:
I think the background information would be helpful as the file name `lahman` may not be self-explanatory (I hope to speak for those who know almost nothing about baseball). I also added a link to the documentation that I found from SeanLahman.com. However, I am not sure whether the documentation correspond to the version of `lahman.db` we have on Scholar.